src/shared/components/DynamicFunctionsList.tsx

```diff
@@ -47,15 +47,15 @@ const DynamicFunctionsList: FC<Props> = ({onSelect}) => {
       event('flux.function.searched', {searchTerm: eventSearchTerm})
       setTermRecorded(eventSearchTerm)
     }
-    setTooltipPopup(false)
```

Comment:
No need to set tooltipPopup to false since tooltip component get unmounted by the time the code reach here